practice2/AboutMePageStyle.css

```diff
@@ -0,0 +1,26 @@
+h1 {
+    font-style:italic;
+    border-style: solid;
+    border-color: red;
+    text-align:center;
+    font-size:3em;
+}
+h2 {
+    padding-top: 20px;
+    padding-bottom: 50px;
+}
+p {
+    font-family: sans-serif;
+    font-size: 2em;
+    width:700px;
+    margin:auto;
+}
+ol {
+    padding:20px;
```

Comment:
you should have a space between the colon and the value ...
look for a feature in the editor (try bringing up the command pallet and typing in "format") that formats the current file ... use that on all your files